src/lib/es2019.array.d.ts

```diff
@@ -17,26 +16,18 @@ interface ReadonlyArray<T> {
      * @param thisArg An object to which the this keyword can refer in the callback function. If
      * thisArg is omitted, undefined is used as the this value.
      */
-    flatMap<U, This = undefined> (
-        callback: (this: This, value: T, index: number, array: T[]) => U | ReadonlyArray<U>,
-        thisArg?: This
-    ): U[]
-
+    flatMap<U>(callback: (value: T, index: number, array: T[]) => U | ReadonlyArray<U>, thisArg?: any): U[];
 
     /**
      * Returns a new array with all sub-array elements concatenated into it recursively up to the
      * specified depth.
      *
      * @param depth The maximum recursion depth
      */
-    flat<A, D extends number = 1>(
-        this: A,
-        depth?: D
-    ): FlatArray<A, D>[]
+    flat<A, D extends number = 1>(this: A, depth?: D): FlatArray<A, D>[];

```

Comment:
Since the rest of this file is formatting changes only, please revert the whole file.